tinywallet/tinywallet/qutilities.py

```diff
@@ -328,15 +326,17 @@ def set(self, on):
 
 def makeWidget(widgetClass, layoutDirection="vertical", parent=None):
     """
-    The creates a tuple of (widget, layout), with layout of type specified with
+    Create a tuple of (widget, layout), with layout of type specified with
     layout direction.
     layout's parent will be widget. layout's alignment is set to top-left, and
-    margins are set to 0 on both layout and widget
+    margins are set to 0 on both layout and widget.
 
-    widgetClass (QtWidgets.QAbstractWidget:) The type of widget to make.
-    layoutDirection (str): optional. default "vertical". One of
-        ("vertical","horizontal","grid"). Determines the type of layout applied
-        to the widget.
+    Args:
+        widgetClass (QtWidgets.QAbstractWidget): The type of widget to make.
+        layoutDirection (str): optional. default "vertical". One of
+            ("vertical","horizontal","grid"). Determines the type of layout
+            applied to the widget.
```

Comment:
Maybe point to the module constants `VERTICAL`, `HORIZONTAL`, and `GRID` instead?